ivy/__init__.py

```diff
@@ -435,11 +435,11 @@ def assert_has_rank(self, rank):
         if self.rank not in (None, rank):
             raise ValueError("Shape %s must have rank %d" % (self, rank))
 
-    def unknown_shape(rank=None, **kwargs):
```

Comment:
I guess it is reasonable to have this function as static. As it produce `unknown_shape` from `rank`